### ISSUE TITLE
Rotate in La*b* space

### ISSUE THREAD

**Issue by StyXman** [sharkdp/pastel#152]
What says on the tin. Very useful when trying to find colors of the same Luminance.

**Comment by sharkdp** [comment]
I agree, that should be supported. Note that this might result in colors outside of the sRGB gamut though. We need a general strategy on how to handle such scenarios.

~~Note that you can already do this with some scripting.~~ You can first extract the current LCh hue with
```
hue=$(pastel format lch-hue "$color")
```
~~and then set the rotated hue with~~
```
pastel set lch-hue "$new_hue"
```

Unfortunately, I just noticed that `lch-hue` can not be `set` yet.
